### DISCUSSION TITLE
CIBA Flow

### DISCUSSION THREAD

**Discussion by nikzen** [zitadel/zitadel#4258] [Ideas]
Hi, I just want to ask if you already support the Client-Initiated Backchannel Authentication Flow or have plans to support it? 

Link to Spec: https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html

Thanks a lot 

Best,

Niklas

**Comment by fforootd** [comment]
Hi, at the moment we have not (yet) planed to support CIBA.
There is also an interest in the[ device authorization grant](https://github.com/zitadel/oidc/issues/141) which has a somewhat overlapping functionality

**Reply by tz-torchai** [reply]
Hi @fforootd , thanks for your reply. 

Could you please re-consider this feature? I think this feature now becomes a high-priority and important for AI agent authentication. 

Auth0 lists AI agent as it's 1st use case of CIBA: https://auth0.com/docs/get-started/authentication-and-authorization-flow/client-initiated-backchannel-authentication-flow#use-cases

We have our own notification infrastructure. This features will integrate well.
